### PR TITLE
General Fixes

### PR DESCRIPTION
**Pull request by ElementalCrisis**
**General** 

- Added fix for Autofill CSS, tested on Chromium and Firefox.

Dashboard

- Changed Wording for better clarification. 
- Changed Series Title font-size to match Collection.
- Changed margin spacing to match Collection.
- Changed opacity on Episode, Amount of Files and Airing Date.

**Collection - Poster View**

- Change Series Font Size
- Increase Margin
- Add Check for Group Count and only show when greater than 1. (Solves last issue in #474.)
- Fix font weight for Missing Image header.

### CODE REVIEW

File: src/pages/collection/items/GridViewGroupItem.tsx [hunk 1]
```diff
@@ -25,20 +25,21 @@ const CountIcon = ({ className, children, show = true }) => (
 const GridViewGroupItem = (item: CollectionGroupType) => {
   const posters = item.Images.Posters;
   const unwatchedCount = item.Sizes.Local.Episodes + item.Sizes.Local.Specials - item.Sizes.Watched.Episodes - item.Sizes.Watched.Specials;
+  const groupCount = item.Sizes.SeriesTypes.Movie + item.Sizes.SeriesTypes.OVA + item.Sizes.SeriesTypes.Other + item.Sizes.SeriesTypes.TV + item.Sizes.SeriesTypes.TVSpecial + item.Sizes.SeriesTypes.Unknown + item.Sizes.SeriesTypes.Web;
 
   return (
     <div key={`group-${item.IDs.ID}`} className="group mr-4 last:mr-0 shrink-0 w-[13.875rem] font-open-sans content-center flex flex-col">
-      <BackgroundImagePlaceholderDiv imageSrc={`/api/v3/Image/${posters[0].Source}/Poster/${posters[0].ID}`} className="h-[19.875rem] rounded drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] border border-black my-2 relative" hidePlaceholderOnHover={true}>
+      <BackgroundImagePlaceholderDiv imageSrc={`/api/v3/Image/${posters[0].Source}/Poster/${posters[0].ID}`} className="h-[19.875rem] rounded drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] border border-black my-2 mb-3 relative" hidePlaceholderOnHover={true}>
```
Comment (harshithmohan) [inline src/pages/collection/items/GridViewGroupItem.tsx:32]: if you are specifying `mb-3`, also change `my-2` to `mt-2` instead

Comment (ElementalCrisis) [reply src/pages/collection/items/GridViewGroupItem.tsx:32]: Whoops, meant to remove ``my-2`` altogether.

File: src/pages/collection/items/GridViewGroupItem.tsx [hunk 2]
```diff
@@ -25,20 +25,21 @@ const CountIcon = ({ className, children, show = true }) => (
...
 
   return (
     <div key={`group-${item.IDs.ID}`} className="group mr-4 last:mr-0 shrink-0 w-[13.875rem] font-open-sans content-center flex flex-col">
-      <BackgroundImagePlaceholderDiv imageSrc={`/api/v3/Image/${posters[0].Source}/Poster/${posters[0].ID}`} className="h-[19.875rem] rounded drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] border border-black my-2 relative" hidePlaceholderOnHover={true}>
+      <BackgroundImagePlaceholderDiv imageSrc={`/api/v3/Image/${posters[0].Source}/Poster/${posters[0].ID}`} className="h-[19.875rem] rounded drop-shadow-[0_4px_4px_rgba(0,0,0,0.25)] border border-black mb-3 relative" hidePlaceholderOnHover={true}>
         <div className="absolute right-0 top-0 group-hover:hidden">
           <CountIcon show={unwatchedCount > 0} className="bg-highlight-2">{unwatchedCount}</CountIcon>
-          <CountIcon className="bg-highlight-5">{item.Size}</CountIcon>
+          <CountIcon show={groupCount >= 2} className="bg-highlight-5">{item.Size}</CountIcon>
```
Comment (revam) [inline src/pages/collection/items/GridViewGroupItem.tsx:35]: Why not `groupCount > 1`?

Comment (ElementalCrisis) [reply src/pages/collection/items/GridViewGroupItem.tsx:35]: Either way works, I just put two in for testing and didn't change it after.

**Comment by ElementalCrisis** [comment]
Added fix for Autofil CSS, tested on Chromium and Firefox.

**Review by hidden4003 — APPROVED** [review]

**Pull request merged**